### PR TITLE
BUGFIX When configured env variables arrive as string, should be int

### PR DESCRIPTION
**Pull request by grubolsch**
### Fixed
Related to the transactional email feature, when the months for the expiration email are configured, it throws an error because env() returns the values as string, but I expect them as integer.

### CODE REVIEW

**Review by JonasVHG — COMMENTED** [review]
I think it might also be a good idea to add `MAILJET_EXPIRATION_TIMER_*` values to `.env.ci`?

File: config/mailjet.php [hunk 1]
```diff
@@ -14,10 +14,10 @@
         'secret' => env('MAILJET_API_SECRET'),
     ],
     'expiration_timers' => [ // Timer is always in months
-        IntegrationType::EntryApi->value => env('MAILJET_EXPIRATION_TIMER_ENTRY_API', 6),
-        IntegrationType::SearchApi->value => env('MAILJET_EXPIRATION_TIMER_SEARCH_API', 6),
-        IntegrationType::UiTPAS->value => env('MAILJET_EXPIRATION_TIMER_UITPAS', 6),
-        IntegrationType::Widgets->value => env('MAILJET_EXPIRATION_TIMER_WIDGETS', 3),
+        IntegrationType::EntryApi->value => (int)env('MAILJET_EXPIRATION_TIMER_ENTRY_API', 6),
```
Comment (LucWollants) [inline config/mailjet.php:17]: I'm not sure if this is the best place to fix this, Laravel has built-in support for this https://laravel.com/docs/11.x/configuration#accessing-configuration-values

It is advised to use `Config::integer`

Comment (grubolsch) [reply config/mailjet.php:17]: :-O O, that is very good to know, I will change it!

Comment (grubolsch) [reply config/mailjet.php:17]: Oké, now I look into it, while this is good knowledge, I am afraid I cannot use it here.
I don't directly access each variable with Config::get(), I pass the entire array.

This gives me a lot more flexibility for in the future if for example we have to not sent mails for one specific type, or quickly add a type.
I would like to keep it this way.

Alternatively I can do a cast to int in app/Console/Commands/SearchExpiredIntegrations.php if you would prefer this.

Comment (LucWollants) [reply config/mailjet.php:17]: Indeed valid remark! It's fine to just keep the `int` conversion on the `env` function

**Review by LucWollants — APPROVED** [review]

**Review by JonasVHG — APPROVED** [review]

**Pull request merged**